docs/wrappers/ledgerutil_postscript.md

```diff
@@ -29,4 +33,19 @@ Here is an example of the `ledgerutil compare` command.
 
   * Note that both snapshot locations must be accessible by the command, for example by mounting volumes from two different peers, or by copying the snapshots to a common location.
 
+### ledgerutil identifytxs example
+
+Here is an example of the `ledgerutil identifytxs` command.
+
+  * Identify relevant transactions in mychannel from a block store within an allowed block range.
+
+    ```
+    ledgerutil identifytxs compare_output/mychannel_5_comparison/first_diffs_by_height.json -o identifytxs_output
+
+    Successfully ran identify transactions tool. Transactions were checked between blocks 1 and 4.
+    ```
+
+    The response above indicates that the local block store was successfully searched. This means all transactions that wrote to keys found in the output JSON of the compare command were identified. The response above also indicates that, if the output JSON was untampered, the highest written key from the output JSON was in block 4, and if the output JSON was edited, that the user limited the search to block 4. In the newly created directory identifytxs_output, a directory mychannel_identified_transactions was generated containing a JSON file of identified transactions for each key from the compare command JSON output.
```

Comment:
"This means all transactions that wrote to keys found in the output JSON of the compare command were identified." Well... just because it was successful doesn't mean that it found ALL of them (it should have, but success doesn't imply it").

"The response above also indicates that, if the output JSON was untampered, the highest written key from the output JSON was in block 4, and if the output JSON was edited, that the user limited the search to block 4."
I think you could remove all this... it is a bit confusing especially the part about "tampered" files.